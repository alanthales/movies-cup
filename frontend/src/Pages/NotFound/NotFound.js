import React from "react";
import Layout from "../../Components/Layout/Layout";

const NotFound = () => (
  <Layout title="404">
    <p className="text-center">Opps! Página não encontrada...</p>
  </Layout>
);

export default NotFound;