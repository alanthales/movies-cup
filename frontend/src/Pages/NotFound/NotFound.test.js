import React from "react";
import NotFound from "./NotFound";
import renderer from "react-test-renderer";

test("render NotFound without errors", () => {
  const component = renderer.create(
    <NotFound />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})