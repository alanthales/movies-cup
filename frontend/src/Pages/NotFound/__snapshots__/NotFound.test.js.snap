// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NotFound without errors 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiBox-root MuiBox-root-105"
    >
      <div
        className="MuiGrid-root header-component MuiGrid-container MuiGrid-justify-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
        >
          <h4>
            CAMPEONATO DE FILMES
          </h4>
          <h1>
            404
          </h1>
          <hr
            style={
              Object {
                "width": "50px",
              }
            }
          />
          <p />
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <p
      className="text-center"
    >
      Opps! Página não encontrada...
    </p>
  </div>
</div>
`;
